tests/storages_tests/rdb_tests/test_with_server.py

```diff
@@ -0,0 +1,98 @@
+from multiprocessing import Pool
+import os
+from typing import Sequence
+from typing import Tuple
+
+import numpy as np
+import pytest
+
+import optuna
+
+_STUDY_NAME = "_test_multiprocess"
+
+
+def f(x: float, y: float) -> float:
+    return (x - 3) ** 2 + y
+
+
+def objective(trial: optuna.Trial) -> float:
+    x = trial.suggest_float("x", -10, 10)
+    y = trial.suggest_float("y", -10, 10)
+    trial.report(x, 0)
+    trial.report(y, 1)
+    trial.set_user_attr("x", x)
+    trial.set_system_attr("y", y)
+    return f(x, y)
+
+
+def run_optimize(args: Tuple[str, str]) -> None:
+    study_name = args[0]
+    storage = args[1]
+    # Create a study
+    study = optuna.create_study(study_name=study_name, storage=storage, load_if_exists=True,)
+    # Run optimization
+    study.optimize(objective, n_trials=20)
+
+
+@pytest.fixture
+def storage() -> str:
+    if "TEST_DB_URL" not in os.environ:
+        pytest.skip("This test requires TEST_DB_URL.")
+    storage_url = os.environ["TEST_DB_URL"]
+    try:
+        optuna.study.delete_study(_STUDY_NAME, storage_url)
+    except KeyError:
+        pass
+    return storage_url
+
+
+def _check_trials(trials: Sequence[optuna.trial.FrozenTrial]) -> None:
+    # Check trial states.
+    assert all((trial.state == optuna.trial.TrialState.COMPLETE for trial in trials))
+
+    # Check trial values and params.
+    assert all(("x" in trial.params for trial in trials))
+    assert all(("y" in trial.params for trial in trials))
```

Comment:
I think you can skip the extra parenthesis(es).
```suggestion
    assert all("y" in trial.params for trial in trials)
```

Reply:
Thank you for your suggestion. I also applied the same changes to the other assertions in https://github.com/optuna/optuna/pull/1497/commits/f9a2d2f74e2bf4e5bd204ce22794a6b264afb777.